docs/tutorials/field_type/7_add_content_and_edit_views.md

```diff
@@ -1,275 +1,116 @@
 # Add content and edit views
```

Comment:
This whole step in v2 is no longer about adding content and edit views, but it is about allowing add/edit form to be handled properly. Is that correct @ViniTou ? 
If so, then we should consider changing the name of the whole step.

Reply:
Yup, that would be more specific indeed.

Reply:
@DominikaK @mere-teresa we need a decision about this. My suggestion would be to go for something like "Allow to add and edit Field in AdminUI", but perhaps better phrased :)

Reply:
`Enable editing the Field in AdminUI` or `in the Back Office` would be my suggestion